sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -58,6 +100,85 @@ def _wrapper_get_presence(physical_port):
     return y_cable_platform_sfputil.get_presence(physical_port)
 
 
+def _wrapper_get_transceiver_info(physical_port):
+    if y_cable_platform_chassis is not None:
+        try:
+            return y_cable_platform_chassis.get_sfp(physical_port).get_transceiver_info()
```

Comment:
For simulated y-cable, the physical cable connected to dual ToR is none-y-cable. But we need xcvrd to initialize instances of simulated y-cable class. To workaround this, we may need to patch this file for testbeds using simulated y-cable. Assume we have simulated y cable at location `sonic-y-cable/microsoft/y_cable_simulated.py`, then can we patch here like below before running tests:
```suggestion
            xcvr_info = y_cable_platform_chassis.get_sfp(physical_port).get_transceiver_info()
            xcvr_info['vendor'] = 'microsoft'
            xcvr_info['model'] = 'simulated'
            return xcvr_info
```

Will this cause any negative impact to the existing xcvr daemon?

Reply:
Would this patch be enabled when running tests or would it be for all cases. 
I think this patch should only be enabled when their is mux_simulator injection code is enabled.

Reply:
This patch will only be injected for dualtor testbed using mux-simulator.

Reply:
Yeah, i think this should work then.